docs: add release notes for dotnet, java, python (#15863)
microsoft/playwright: docs/src/release-notes-python.md

<!-- TOC -->

## Version 1.24

### 🐂 Debian 11 Bullseye Support

Playwright now supports Debian 11 Bullseye on x86_64 for Chromium, Firefox and WebKit. Let us know
if you encounter any issues!

Linux support looks like this:

|          | Ubuntu 18.04 | Ubuntu 20.04 | Ubuntu 22.04 | Debian 11
| :--- | :---: | :---: | :---: | :---: | 
| Chromium | ✅ | ✅ | ✅ | ✅ |
| WebKit | ✅ | ✅ | ✅ | ✅ |
| Firefox | ✅ | ✅ | ✅ | ✅ |

### New introduction docs

> We're rewrote our docus to be more end-to-end testing focused, see on [playwright.dev](https://playwright.dev/python/docs/intro).

Reviewer: ```suggestion
We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).
```

> We rewrote our docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).

Reviewer: ```suggestion
We rewrote our _Getting Started_ docs to be more end-to-end testing focused. Check them out on [playwright.dev](https://playwright.dev/python/docs/intro).
```
